project/Dependencies.scala

```diff
@@ -81,8 +81,8 @@ object Dependencies {
   val cats: ModuleID =            "org.typelevel"                 %% "cats-core"                 % "2.6.1"
   val logbackClassic: ModuleID =  "ch.qos.logback"                % "logback-classic"       % "1.2.10"
   val scalaUri: ModuleID =        "io.lemonlabs"                  %% "scala-uri"            % "3.0.0"
-  val scalatest: ModuleID =       "org.scalatest"                 %% "scalatest"            % "3.2.2" % "test"
-  val mockito: ModuleID =         "org.scalatestplus"             %% "mockito-3-4"          % "3.2.10.0" % Test
+  val scalatest: ModuleID =       "org.scalatest"                 %% "scalatest"            % "3.2.11" % "test"
+  val mockito: ModuleID =         "org.scalatestplus"             %% "mockito-4-2"          % "3.2.11.0" % Test
```

Comment:
had to upgrade mockito because the `mockito-3-4` has errors trying to mock certain classes under java 17. I went with the latest `org.scalatestplus.mockito-4-2`, which indicated a corresponding upgrade of scalatest itself to 3.2.11.